cartography/models/aws/rds/snapshot.py

```diff
@@ -0,0 +1,108 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class RDSSnapshotNodeProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef("DBSnapshotArn")
+    arn: PropertyRef = PropertyRef("DBSnapshotArn", extra_index=True)
+    db_snapshot_identifier: PropertyRef = PropertyRef(
+        "DBSnapshotIdentifier", extra_index=True
+    )
+    db_instance_identifier: PropertyRef = PropertyRef("DBInstanceIdentifier")
+    snapshot_create_time: PropertyRef = PropertyRef("SnapshotCreateTime")
+    engine: PropertyRef = PropertyRef("Engine")
+    allocated_storage: PropertyRef = PropertyRef("AllocatedStorage")
+    status: PropertyRef = PropertyRef("Status")
+    port: PropertyRef = PropertyRef("Port")
+    availability_zone: PropertyRef = PropertyRef("AvailabilityZone")
+    vpc_id: PropertyRef = PropertyRef("VpcId")
+    instance_create_time: PropertyRef = PropertyRef("InstanceCreateTime")
+    master_username: PropertyRef = PropertyRef("MasterUsername")
+    engine_version: PropertyRef = PropertyRef("EngineVersion")
```

Comment:
nit again: would be good to put this line with engine_version on line 24, underneath engine